app/plugins.go

```diff
@@ -19,63 +19,96 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/utils"
 
-	"github.com/mattermost/mattermost-server/app/plugin"
+	builtinplugin "github.com/mattermost/mattermost-server/app/plugin"
```

Comment:
Relabeling the pre-existing stuff as "built-in" plugins. Once we port the JIRA / LDAP stuff, we can get rid of this altogether.